packages/contracts-bedrock/src/L1/OPStackManager.sol

```diff
@@ -151,44 +175,45 @@ contract OPStackManager is ISemver {
 
     // -------- Methods --------
 
-    /// @notice OPSM is intended to be proxied when used in production. Since we are initially
-    /// focused on an OPSM version that unblocks interop, we are not proxying OPSM for simplicity.
-    /// Later, we will `_disableInitializers` in the constructor and replace any constructor logic
-    /// with an `initialize` function, which will be a breaking change to the OPSM interface.
-    constructor(SuperchainConfig _superchainConfig, ProtocolVersions _protocolVersions, Blueprints memory _blueprints) {
-        // TODO uncomment these as we add more validations to this contract, currently this will break a few tests.
-        // assertValidContractAddress(address(_superchainConfig));
-        // assertValidContractAddress(address(_protocolVersions));
-        // assertValidContractAddress(_blueprints.addressManager);
-        // assertValidContractAddress(_blueprints.proxy);
-        // assertValidContractAddress(_blueprints.proxyAdmin);
-        // assertValidContractAddress(_blueprints.l1ChugSplashProxy);
-        // assertValidContractAddress(_blueprints.resolvedDelegateProxy);
-
+    /// @notice OPSM is proxied. Therefore the `initialize` function replaces any constructor logic for this contract.
```

Comment:
nit, since we still are setting immutables here
```suggestion
    /// @notice OPSM is proxied. Therefore the `initialize` function replaces most constructor logic for this contract.
```

Reply:
Updated in: https://github.com/ethereum-optimism/optimism/pull/11875/commits/a77e799e4e37389132993e8883cd14678ff78d23#diff-adc000b158932be783e0ea3e4fac762b4aa8a6310d4088675627630dc977eaddR171